internal/resource/config/administrativeaccount/administrative_account_resource.go

```diff
@@ -75,49 +75,52 @@ func (r *administrativeAccountsResource) Schema(ctx context.Context, req resourc
 				Optional:    true,
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.UseStateForUnknown(),
-					stringplanmodifier.RequiresReplace(),
 				},
 			},
 			"description": schema.StringAttribute{
 				Description: "Description of the account.",
 				Optional:    true,
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.UseStateForUnknown(),
-					stringplanmodifier.RequiresReplace(),
 				},
 			},
 			"email_address": schema.StringAttribute{
 				Description: "Email address associated with the account.",
 				Optional:    true,
-				Computed:    true,
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.UseStateForUnknown(),
-					stringplanmodifier.RequiresReplace(),
 				},
 			},
 			"password": schema.StringAttribute{
-				Description: "Password for the Account. This field is only applicable during a POST operation.",
-				Required:    true,
+				Description: "Password for the Account. This field is only applicable during account creation.",
+				Computed:    true,
+				Optional:    true,
+				Sensitive:   true,
+				PlanModifiers: []planmodifier.String{
+					stringplanmodifier.RequiresReplaceIfConfigured(),
+				},
+			},
+			"encrypted_password": schema.StringAttribute{
+				Description: "Read-only attribute. This field is utilized for updating an existing Administrative Account.",
```

Comment:
Might want to mention in the description that this field holds the encrypted password returned by ping fed

Reply:
Description has been updated